docsrc/imap/developer/major-releasing.rst

```diff
@@ -0,0 +1,415 @@
+   b. Make the directories for the new dev version:
+      ``mkdir -p docsrc/imap/download/release-notes/<dev>/x``
+   c. Create `docsrc/imap/download/release-notes/<version>/index.rst`
+      for each of these, with stub contents.  It's easiest to just
+      copy and update from an older one.
+   d. Add stub release notes for alpha0.  This will be a file called
+      `docsrc/imap/download/release-notes/<series>/x/<series>.0-alpha0.rst`.
+      It's easiest to copy the release notes file from the previous major
+      release, delete all the bullet points (leaving just the headings), and
+      fix all the numbers.
+
+7. Update `README.md`:
+
+   - It will be claiming to be the development version, but this is now (or
+     will be) the stable version, so update that.  If in doubt, mimick what
+     the old stable branch's copy says.  This is another set of lies that will
+     become true as we go.
+   - Search through the whole document for version numbers, and update them
+     sensibly for the future reality.  Do this mindfully, not with a batch
+     find-replace.
+   - This is also a good time for a careful review of the contents of this
+     file.  Fix anything that's out of date, missing, etc.
+
+8. Make sure your RST changes are good:  ``make doc-html``.  Pay attention
+   to any errors or warnings (they will be coloured).  There will be some
+   you can clearly ignore, such as glob patterns for future release notes
+   that don't exist yet, but do your best to deal with everything else.
+   The generated documentation will be under the `doc/html/` directory --
+   examine it in your browser to make sure all your formatting and such makes
+   sense.
+9. Tell Github Actions about the new branch: edit `.github/workflows/main.yml`
+   and add the new branch to the list in the obvious spot.  Does anything else
+   need to be done for this step?  Unknown... figure it out and document it!
+   Also currently unknown whether this needs to happen on the master branch,
+   on the branch itself, or both.  So we do it on both just in case.
+10. XXX maybe missing some stuff here still?
+
+You can double check your work by looking at what changed last time a new
+stable series was forked:
+``git show --format=fuller cyrus-imapd-<oldstable>.0-alpha0``.
+Also look a few commits forward, in case the previous releaser
+missed steps before tagging, and had to catch them up later.
+
+Once you're satisfied that you've done everything that needed doing here:
+
+1. Commit all these changes.  A single commit is good, we would like this to
+   be the very first commit after the fork point.
+2. Create a signed, annotated tag declaring that this is now alpha0 of the .0
+   release of the new series:
+   ``git tag -s cyrus-imapd-<series>.0-alpha0``
+3. You will be prompted to enter a commit message for the tag (this is
+   what makes it an "annotated" tag).  Ellie uses something like "not a real
+   release, but need a tag for versioning".
```

Comment:
Let's pick an actual suggested string, then we can put it in the `-m` argument to be copypasted.  Maybe: "Initial (non-released) tag of <series> releases"